AptosSDK/Src/Account.cs

```diff
@@ -0,0 +1,61 @@
+using Chaos.NaCl;
+using Mirage.Aptos.Utils;
+using Org.BouncyCastle.Crypto.Digests;
+
+namespace Mirage.Aptos.SDK
+{
+	public class Account
+	{
+		private const byte AddressEnding = 0x00;
+		
+		private byte[] _publicKey;
+		private byte[] _privateKey;
+		
+		public readonly string Address;
+		public readonly string PublicKey;
+
+		public Account()
+		{
+			var seed = GenerateRandomSeed();
+			CreateKeyPairFromSeed(seed);
+			Address = GetAddress();
+			PublicKey = _publicKey.ToHex(true);
+		}
+
+		public Account(byte[] privateKey)
+		{
+			var seed = privateKey.Slice(0, 32);
+			CreateKeyPairFromSeed(seed);
+			Address = GetAddress();
+			PublicKey = _publicKey.ToHex(true);
+		}
+
+		private static byte[] GenerateRandomSeed()
+		{
+			byte[] bytes = new byte[Ed25519.PrivateKeySeedSizeInBytes];
+			RandomUtils.GetBytes(bytes);
+			return bytes;
+		}
+
+		private void CreateKeyPairFromSeed(byte[] seed)
+		{
+			_privateKey = Ed25519.ExpandedPrivateKeyFromSeed(seed);
+			_publicKey = Ed25519.PublicKeyFromSeed(seed);
+		}
+
+		public string GetAddress()
```

Comment:
please fix method orders in SRC folder. private goes below public. Refer to code style.